### DISCUSSION TITLE
How can I run slint ui without windowing systems, just as QT Eglfs backend, supports GLES, DRM/KMS

### DISCUSSION THREAD

**Discussion by fluolite** [slint-ui/slint#5886] [General] [answered]
Hi !
How can I run slint ui without windowing systems, just as QT Eglfs backend, supports GLES, DRM/KMS

**Comment by ogoffart** [comment]
You can use the linuxkms backend.
You need to enable the backend with the feature flag.
More info there: https://releases.slint.dev/1.7.2/docs/slint/src/advanced/backend_linuxkms.html

**Reply by fluolite** [reply]
Thx!
Can I open multiple window with diffrent special monitor, or bind crtc or plane for renderer output?

**Reply by tronical** [reply, marked as answer]
At the moment only single-window-single-drm-output is implemented. This could of course be extended to support different outputs per window. The same goes for the plane selection. It would be a nice feature to have.

**Reply by fluolite** [reply]
Thx,  I am very look forward for it's new feature.

**Reply by aurindam** [reply]
@fluolite If this issue is a blocker for you, please let us know by [email](mailto:info@slint.dev) or [chat](https://chat.slint.dev/). Share details about your project, so that we can prioritise it or offer it as a service.
